R/browse_metadata.R

```diff
@@ -153,7 +153,7 @@ browse_metadata <- function(json_file = NULL, output_dir = NULL) {
   # SAVE PLOTS ----
 
   original_wd <- getwd()
-  setwd(output_dir) # saveWidget has a bug with paths & saving
+  on.exit(output_dir) # saveWidget has a bug with paths & saving
```

Comment:
I *think* this line stays the same? I haven't looked into on.exit too much but I think your change on line 171 is correct, but not here

Reply:
Yes, example files will be awesome!!